test/Kestrel.Transport.BindTests/HandleInheritanceTest.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using System.IO;
+using System.Net.Sockets;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.AspNetCore.Testing.xunit;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.FunctionalTests
+{
+    [OSSkipCondition(OperatingSystems.MacOSX)] // Not applicable, and vi doesn't work there.
```

Comment:
Are we sure handle inheritance isn't an issue on macOS?

Reply:
Yes, the corefx issue was clear that this is only an issue for Windows.